packages/letters-of-credit/src/components/DetailsCard/DetailsCard.js

```diff
@@ -53,9 +53,9 @@ class DetailsCard extends Component {
             <span class="subheadingSpan">{this.state.data[1]}</span>
             <span class="subheadingSpan, topHeading">COMPANY NAME</span>
             <span class="subheadingSpan">{this.state.data[2]}</span>
-            <span class="subheadingSpan, topHeading">SORT CODE</span>
-            <span class="subheadingSpan">{this.state.data[3]}</span>
             <span class="subheadingSpan, topHeading">ACCOUNT NUMBER</span>
```

Comment:
There should be no account number if we're using IBAN as sort code/acc no go together.